delivery_postlogistics/__manifest__.py

```diff
@@ -0,0 +1,29 @@
+# © 2013-2016 Yannick Vaucher (Camptocamp SA)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "name": "PostLogistics Shipping - “Barcode” web service",
+    "summary": "Print PostLogistics shipping labels using the Barcode web service",
+    "version": "18.0.1.0.0",
+    "author": "Camptocamp,Odoo Community Association (OCA)",
+    "maintainer": "Camptocamp",
+    "license": "AGPL-3",
+    "category": "Delivery",
+    "complexity": "normal",
+    "depends": ["stock_delivery"],
```

Comment:
```suggestion
    "depends": ["stock_delivery", "delivery_carrier_shipping_label"],
```
Please, we need to be closer to the future label management. (Several other delivery carriers implementation will benefits from it)